fixtures/stripe-fixtures.json

```diff
@@ -0,0 +1,75 @@
+{
+    "_meta": {
+        "template_version": 0
+    },
+    "fixtures": [
+        {
+            "name": "prod_hobby",
+            "path": "/v1/products",
+            "method": "post",
+            "params": {
+                "name": "Hobby",
+                "description": "Hobby product description",
+                "metadata": {
+                    "id": "prod_hobby"
+                }
+            }
+        },
+        {
+            "name": "prod_freelancer",
+            "path": "/v1/products",
+            "method": "post",
+            "params": {
+                "name": "Freelancer",
+                "description": "Freelancer product description",
+                "metadata": {
+                    "id": "prod_freelancer"
+                }
+            }
+        },
+        {
+            "name": "plan_hobby",
+            "path": "/v1/plans",
```

Comment:
Plans are outdated, can you change to prices instead, please? https://stripe.com/docs/api/prices